amundsen_application/static/js/components/SearchPage/tests/index.spec.tsx

```diff
@@ -606,6 +637,50 @@ describe('SearchPage', () => {
     });
   });
 
+  describe('renderContent', () => {
+    let content;
```

Comment:
I see you're not actually using the `content` variable in your tests so we can remove those.